### PR TITLE
13.0 mig product pricelist brand

### PR DESCRIPTION
**Pull request by feketemihai**

### CODE REVIEW

File: product_pricelist_brand/models/product_pricelist.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+from odoo import api, fields, models, tools, _
+from odoo.exceptions import UserError
+
+
+class ProductPricelist(models.Model):
+    _inherit = 'product.pricelist'
+
+    @api.multi
+    def _compute_price_rule(self, products_qty_partner, date=False,
```
Comment (qgroulard) [inline product_pricelist_brand/models/product_pricelist.py:12]: This looks like a lot of duplicated code to me.
The base code is definitely not meant to be extended. In my version I override `_compute_price_rule_get_items`, I don't really like it neither but still I think it is a bit cleaner.
Unless there is a good reason to  work on `_compute_price_rule` that I am missing ?

File: product_pricelist_brand/models/product_pricelist.py [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+                 'compute_price', 'fixed_price', 'pricelist_id',
+                 'percent_price', 'price_discount', 'price_surcharge')
+    def _get_pricelist_item_name_price(self):
+        super(ProductPricelistItem, self)._get_pricelist_item_name_price()
+        for item in self:
+            if item.brand_id:
+                item.name = _("Brand: %s") % (item.brand_id.name)
+
+    @api.onchange('applied_on')
+    def _onchange_applied_on(self):
```
Comment (qgroulard) [inline product_pricelist_brand/models/product_pricelist.py:136]: In the base module there is no `_onchange_applied_on` but updates in create and write, it is more robust to stick to that.
Plus don't forget the method `_check_product_consistency`.

**Review by qgroulard — CHANGES_REQUESTED** [review]
Well, I don't know how but I didn't see this PR and opened https://github.com/OCA/brand/pull/44.
I guess we can get the better of both PRs so please have a look at https://github.com/OCA/brand/pull/44.

First, please follow the migration guidelines (https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#howto) in order to keep the history and solve formatting issues.
I can't find any trace of this module (here nor in repo product-attribute).

Here are my comments, I'm looking forward to reading your suggestions.

**Comment by sbejaoui** [comment]
@feketemihai , any news about this PR?

**Pull request closed without merging**